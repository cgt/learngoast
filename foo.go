package learngoast

import "fmt"

func IsApplesauce(x string) bool {
	return x == "applesauce"
}

func PrintIfApplesauce(x string) {
	if IsApplesauce(x) {
		fmt.Println("It is applesauce!")
	}
}
