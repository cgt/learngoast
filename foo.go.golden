package learngoast

import "fmt"

func renamed(x string) bool {
	return x == "applesauce"
}

func PrintIfApplesauce(x string) {
	if renamed(x) {
		fmt.Println("It is applesauce!")
	}
}
